### PR TITLE
additional information for "Migrate LDAP IdAttribute"

### PR DESCRIPTION
**Pull request by JtheBAB**
#### Summary
The explanation for mmctl ldap id migration misses some important background information compared to [the old command line doc](https://docs.mattermost.com/manage/command-line-tools.html#mattermost-ldap-idmigrate).

Additional there was a small copy & paste error for the -h section.

### CODE REVIEW

**Review by justinegeffen — APPROVED** [review]
This looks good editorially. Thank you!

**Review by cwarnermm — APPROVED** [review]
Thank you so much for this contribution! Next steps include this content being technically reviewed for accuracy.

File: source/manage/mmctl-command-line-tool.rst [hunk 1]
```diff
@@ -2996,7 +2996,12 @@ mmctl ldap idmigrate
 
 **Description**
 
-Migrate LDAP ``IdAttribute`` to a new value. Run this utility then change the IdAttribute to the new value.
+Migrate LDAP ``IdAttribute`` to a new value. Run this utility to change the value of your ID Attribute without your users losing their accounts. After running the command you can change the ID Attribute to the new value in the System Console. For example, if your current ID Attribute was ``sAMAccountName`` and you wanted to change it to ``objectGUID``, you would:
+
+- Wait for an off-peak time when your users won’t be impacted by a server restart.
```
Comment (justinegeffen) [inline source/manage/mmctl-command-line-tool.rst:3001]: If this has to be performed in this sequence, I'm wondering if a numbered list may work better?

Comment (JtheBAB) [reply source/manage/mmctl-command-line-tool.rst:3001]: Up to you :)

Yes it is a sequence

Comment (JtheBAB) [reply source/manage/mmctl-command-line-tool.rst]: should be ok

**Comment by justinegeffen** [comment]
@mgdelacroix, gentle ping on this when you have the bandwidth for a review. Thank you! :)

**Comment by mgdelacroix** [comment]
I'm afraid I'm not familiar with how the migration works under the hood, so I'm adding @mkraft as the tech reviewer instead.

It would be good to add these changes to the `mmctl` interactive docs as well, so I've created https://github.com/mattermost/mmctl/pull/374 to sync them and be merged after this one

**Comment by mkraft** [comment]
Removed myself as a reviewer, as discussed @cwarnermm

**Review by icelander — APPROVED** [review]
LGTM Thanks @JtheBAB!

**Pull request merged**